packages/nimble-components/src/table/index.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '@tanstack/table-core';
```

Comment:
Maybe putting the components into subfolders was a good idea
```
table/components/cell
table/components/row
table/index.ts
table/template.ts
```

Having the github PRs mix around the directories and files is going to make reviews annoying

```
cell/
index.ts
row/
styles.ts
template.ts
```

But I guess column config in the future will stay top-level since the user seems them:

```
table/
table-column-base/    (since we have that pattern with IconBase and AnchorBase now)
table-column-string/
table-column-number/
```

Reply:
I moved the `cell` and `row` folders into a `components` folder as suggested.